Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG centos_version=7
+FROM centos:${centos_version}
+
+ENV REFRESHED_AT=2018-08-16 \
+    LANG=en_US.UTF-8 \
+    HOME=/opt/build \
+    TERM=xterm
+
+WORKDIR /opt/build
+
+ENV HOME /root
+
+RUN \
+  yum -y groupinstall "Development Tools" && \
+  yum -y install gcc gcc-c++ glibc-devel make ncurses-devel openssl-devel autoconf java-1.8.0-openjdk-devel git && \
+  yum -y install wget && \
+  yum -y install epel-release && \
+  yum -y install unzip && \
+  yum -y install npm
+
+# Install asdf
+RUN \
```

Comment:
Also, because each RUN translates to a new docker image layer (commit), you could economize by merging the following runs together.. at the expense of readability 😢